web/modules/custom/do_base/do_base.deploy.php

```
@@ -8,3 +8,225 @@
+    $node->set('field_c_n_summary', 'Whether you need a new Drupal build, help upgrading from an end-of-life version, or ongoing support from a senior team, we are happy to have an honest conversation about where things stand.');
+  }
+
+  foreach ($node->get('field_c_n_components')->referencedEntities() as $existing) {
+    $existing->delete();
+  }
+
+  $webform = Paragraph::create([
+    'type' => 'civictheme_webform',
+    'field_c_p_theme' => 'dark',
+    'field_c_p_webform' => 'contact',
+  ]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Drupal\paragraphs\Entity\Paragraph' in method 'do_base_deploy_contact'. (undefined)

(StaticAccess)